package/MDAnalysis/analysis/dssp.py

```diff
@@ -0,0 +1,279 @@
+from .base import AnalysisBase
+import numpy as np
+from einops import repeat, rearrange
+from .. import Universe
+
+CONST_Q1Q2 = 0.084
+CONST_F = 332
+DEFAULT_CUTOFF = -0.5
+DEFAULT_MARGIN = 1.0
```

Comment:
Do these constants from from the original DSSP algorithm?

Reply:
yes -- see "Hydrogen-Bonded Structure" part of the paper.